### PR TITLE
Use gender inclusive language in comment in nvt_dsc_pps.c

### PR DESCRIPTION
**Pull request by BigFatHairyGamer**
Replaces "poor-man" -> "poor-person"

### CODE REVIEW

**Comment by hendramanudinata03** [comment]
Wow, very useful. 🤯🤯

**Comment by MitchTalmadge** [comment]
I appreciate this PR, personally. Thank you for being inclusive.

**Comment by ment1na** [comment]
> Wow, very useful. 🤯🤯
> 
> I think PRs are supposed to improve the driver?

this drastically improves the driver

**Comment by aenoboa1** [comment]
Massive optimization this

**Comment by StayBlueee** [comment]
This will massively improve the performance of the Nvidia driver (I think).

**Comment by ShiSHcat** [comment]
Major update, thanks for your work

**Comment by xerosic** [comment]
This the commit that will improve the NVIDIA drivers, thank you kind ma- ehm I mean human being(?)

**Comment by himdeez** [comment]
> This the commit that will improve the NVIDIA drivers, thank you kind ma- ehm I mean human being(?)

Hu-"man"?

**Comment by iskamag** [comment]
bruh

**Comment by brandonsaldan** [comment]
its not hacktoberfest yet?

**Comment by isage** [comment]
This isn't inclusive for those, who identify as Apache attack helicopter, you bigot.

**Comment by poyynt** [comment]
I appreciate this and hope for the day when people stop being ignorant about these stuff.

**Comment by arnavbhatt288** [comment]
Can you ~guys~ people stop introducing anglo-centric nonsense and actually create a useful PR?

**Comment by theVakhovskeIsTaken** [comment]
all the best💞 thanks you💞

**Comment by Madouura** [comment]
We finally get something good from NVIDIA and this is what you people do.
This is why I love and hate the internet.

**Comment by tiklii** [comment]
person ❌
perdaughter ✅

**Comment by tiklii** [comment]
2. Man [uncountable] humans as a group, both male and female 
•the damage caused by man to the environment
•The common man (= ordinary people) is losing faith in democratic institutions.

**Comment by bothalissom** [comment]
> 2. Man [uncountable] humans as a group, both male and female
>    •the damage caused by man to the environment
>    •The common man (= ordinary people) is losing faith in democratic institutions.

Shhh! Stop ruining everything with your literate Western bigotry. Maybe your ancestors weren't chimpanzees in Africa, but you could show some respect to the rest of us.

File: src/nvidia-modeset/src/nvkms-rm.c [hunk 1]
```diff
@@ -4983,7 +4983,7 @@ static void MuxInit(const NVDpyEvoRec *pDpyEvo)
          * is disconnected from the GPU. */
 
         /* Map with hard-coded data for systems known to support dynamic mux
-         * switching. This is a poor-man's alternative to the WDDM driver's
+         * switching. This is a poor-person's alternative to the WDDM driver's
```
Comment (muhammednagy) [inline src/nvidia-modeset/src/nvkms-rm.c:4986]:
```suggestion
         * switching. This is a poor-human's alternative to the WDDM driver's
```

**Review by muhammednagy — CHANGES_REQUESTED** [review]

**Comment by ShiSHcat** [comment]
> We finally get something good from NVIDIA and this is what you people do. This is why I love and hate the internet.

We doin a little trolling

File: src/nvidia-modeset/src/nvkms-rm.c (hunk 1 above)
Comment (ShiSHcat) [reply src/nvidia-modeset/src/nvkms-rm.c:4986]: You can't be such disrespectful against non-human persons!

**Comment by Kefta** [comment]
The use of the word poor here is very classist, can we find an alternative that doesn't discriminate along economic lines?

**Comment by MrCyjaneK** [comment]
@Kefta, let's just remove all the comments so nobody will get hurt by dangerous gender misinformation, or will feel hurt by someone's comment to their own code written back in the 90s or early 20s.

Seriously people I waited quite long for this source code. Instead of focusing or things like somebody thoughts over his own code.
And I'll tell you, there is so much hate speech targeted at future/past myself in code I do for work that fixing it would take insane amount of time.

So my suggestion is, for history sake - do not edit comments that are already here, instead if you write some code, document it the way you like. Don't forget that it isn't exactly the source that NVIDIA guys are using in their company, they probably have to manually cherry-pick the commits to their repo, and requiring somebody to go thru all the verification, checks just for one line that doesn't even affect the final binary is a waste of time.

**Comment by shkhln** [comment]
> The use of the word poor here is very classist, can we find an alternative that doesn't discriminate along economic lines?

disadvantaged-tux?

**Comment by BurnyBoi** [comment]
image

**Comment by camatthew** [comment]
this is a stupid pull request

**Comment by aurelienpierre** [comment]
It took 15 years to a million-dollars company to release a proper Linux driver, and only 1 day to the PC lobby to find a bug in its wording.

File: src/nvidia-modeset/src/nvkms-rm.c (hunk 1 above)
Comment (sprunk) [reply src/nvidia-modeset/src/nvkms-rm.c:4986]:
```suggestion
         * switching. This is a poverty-stricken person's alternative to the WDDM driver's
```

**Comment by aurelienpierre** [comment]
My mistake. Bug also implies functional code, not comments.

**Comment by BigFatHairyGamer** [comment]
> It took 15 years to a million-dollars company to release a proper Linux driver, and only 1 day to the PC lobby to find a bug in its wording.

YEAH!!! IM A PC GAMER BROTHER. THATS WHY IM HERE??

Fuck you XBOX or SONY fans

File: src/nvidia-modeset/src/nvkms-rm.c (hunk 1 above)
Comment (BigFatHairyGamer) [reply src/nvidia-modeset/src/nvkms-rm.c:4986]: Perhaps financially-disadvantaged?

**Comment by yesnomaybeyes** [comment]
@aurelienpierre muh PC lobby

it's an obvious joke/troll account, idiot

**Comment by mtijanic** [comment]
Friendly reminder to everyone to please observe our [Code of Conduct](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CODE_OF_CONDUCT.md) and keep all discussions friendly and welcoming.

**Comment by aritger** [comment]
We'll review the code base overall for appropriate language, but closing this particular PR because the exchange here has gotten out of hand.  Please abide by the [Code of Conduct](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CODE_OF_CONDUCT.md) and be kind to one another.  I don't want to have to ban anyone.

**Comment by motoreagattoimburrato** [comment]
437-4377822_and-gooby-gooby-pls-hd-png-download
 no... pls... stap!

**Comment by GabrielleShull** [comment]
> 2. Man [uncountable] humans as a group, both male and female
>        •the damage caused by man to the environment
>        •The common man (= ordinary people) is losing faith in democratic institutions.

I get this pr is not useful in any technical way, and op is almost definitely a troll; however, I think comments like yours bringing up this definition of man are a bit misinformative about what people like me asking for these kinds of change (in places where they actually make more sense) want and understand. We already know the definition of "man" is gender neutral in the context of it as a root word; however, the _connotations_ of "man" when used as a root word are often not gender neutral. The issue with the connotations is especially bad with roles like "fireman" and "policeman" where those positions are often compared with superheroes. Superheroes often have uncreative names that emphasize their genders like "Superman" and "Wonder Woman." Those kinds of comparisons combined with the root words in the real life roles can sometimes give children and illiterate people the wrong impression that masculinity is important for policing and firefighting. We think it's better to use other root words like "person" or noun-phrases like "police officer" to avoid those misleading connotations. That's all.

**Comment by yamirui** [comment]
I'm not a person and I feel excluded by this.

**Comment by BurnyBoi** [comment]
> Friendly reminder to everyone to please observe our [Code of Conduct](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CODE_OF_CONDUCT.md) and keep all discussions friendly and welcoming.

Okay. In the most friendly and welcoming way possible: Nobody wants identity politics in their software. Poor man's [thing] has been a common phrase since forever and OP is deliberately ignoring this for politics (also will never be a woman).

**Comment by GabrielleShull** [comment]
> Are you seriously implying that masculinity isn't important for these jobs? It takes a man's strength to do both of these jobs effectively, and you're only fooling yourself if you actually believe otherwise.

Respectfully, I believe it simply requires strength that people of all healthy sex karyotypes can reasonably attain through nutrition and dedicated training to do both jobs effectively. Additionally, I think empathy, multitasking, and communications skills are important for both of those jobs. While athletic skills often have masculine connotations and social skills often have feminine connotations, I don't believe either are intrinsically masculine or feminine. There are species where males have higher average strength, species where females have higher average strength, and species where strength is not a sexually dimorphic trait. I would not say a female hyena is "masculine" or has "a man's strength." I also wouldn't say that a woman police officer with exemplary strength has a "man's strength" or that a man police officer with exemplary communications skills has a "woman's mouth."

**Comment by proudmuslim-dev** [comment]
Is this really what people are spending their time on these days? Good lord, we're regressing

**Comment by yamirui** [comment]
> Lastly, I think the perception that masculinity is needed

If you want to use the word "think" properly at least think of what people actually mean and not what you just made up. When it comes to law enforcement, you want someone who is both strong and capable of rational thinking, none of which are traits of a woman. And I know this because only a woman would imply that women are capable of enforcing the law. Sure, go ahead, enforce the law, when you're kicked to the curb and most likely killed and 3 male police officers are called in to deal with the problem.

Same applies to firefighters. I don't know what's inside your skull, but it's not a brain, because a woman cannot lift a teenager, let alone an injured man who needs to be taken out of a burning house in seconds.

**Comment by BurnyBoi** [comment]
In conclusion: great job OP for trolling the repo. Clearly you weren't serious yet you got all us in here regardless:

image

**Pull request closed without merging**